subprojects/configuration-cache/src/integTest/groovy/org/gradle/configurationcache/ConfigurationCacheMultiProjectIntegrationTest.groovy

```diff
@@ -16,8 +16,40 @@
 
 package org.gradle.configurationcache
 
+import spock.lang.Issue
+
 class ConfigurationCacheMultiProjectIntegrationTest extends AbstractConfigurationCacheIntegrationTest {
 
+    @Issue("https://github.com/gradle/gradle/issues/18897")
```

Comment:
🎨 I'd expect to find this test in `ConfigurationCacheScriptChangesIntegrationTest` or under a new `ConfigurationCacheMissingScriptIntegrationTest` class.

Reply:
Agree 👍